### DISCUSSION TITLE
MS Defender blocks 'Serious threat' when running log4jscanwin

### DISCUSSION THREAD

**Discussion by hitem** [Qualys/log4jscanwin#34] [General]
Hello,
Your tool is triggering AV's. I tried looking through your code but could not find anything suspicious at a glance, therefor this submit.
What it finds is the following:
![bild](https://user-images.githubusercontent.com/8977898/146657985-5385dcfa-fd49-49f8-80ea-34580a896725.png)
It says: Serious threat, blocked and removed.

Hope it helps.

**Comment by romw** [comment]
Was it triggered from one of the binaries we supplied?  Or did you build it yourself?

**Comment by hitem** [comment]
From your binary. It runs for about 25 seconds "Scanning c:\" and then it comes up.
Edit: i also tried running it on a secondary system and same trigger there. Three times in a row (incase it was something else!).

**Comment by bushy555** [comment]
Useless side note that may or may not help:
Running the scanning tool on an old XP computer (works great even on old XP !). 
Avast with current signatures (20 Dec) does not detect it being a threat.

**Comment by Olivier-CracK** [comment]
Quick feedback : I had no problem in running the program on win 10 with SentinelOne av.

**Comment by johnmccash** [comment]
I had no problems running this (1.2.17) yesterday on Win10 with Defender for Endpoints

**Comment by hitem** [comment]
Thanks for the tests guys. I went ahead and went a little deeper today. 
However, its still detected by Microsoft Defender.
![image](https://user-images.githubusercontent.com/8977898/146795040-704daf78-728e-484b-b86c-38e7b17a4b6b.png)
Tried running w and w/o administrator rights - tried disabling each feature of defender and the only time it was undetected is if i turn defender completely off....

![image](https://user-images.githubusercontent.com/8977898/146795408-776d664b-b6ab-485f-b11d-83211901f49c.png)

**Comment by romw** [comment]
I suspect the ransomware behavior detection component of Windows 11 noticed the sequential traversal of the file system as something to block.  It appears our code-signing certificate isn't enough to overcome the suspicion of Windows Defenders scoring system on Windows 11.

That is rather annoying.

**Comment by gsmith89** [comment]
I am getting a similar message from Defender when running on Windows 11. It  is saying the program is putting a file in the temp folder of AppData.
![image](https://user-images.githubusercontent.com/30840864/146814835-2b0489f8-88d9-4e5b-be74-9d9d85e9910a.png)

**Comment by pjheslop** [comment]
Symantec Endpoint Protection also blocks the tool: 
![image](https://user-images.githubusercontent.com/62217785/147254880-17ea21ef-98d1-44b1-919c-0dcff5bf292f.png)
